tests/ccgbugs/tcodegen_anonymous_aggregate.nim

```diff
@@ -0,0 +1,34 @@
+discard """
+targets: "c cpp"
+labels: "backend_cpp , pragma , atomics , union"
```

Comment:
Backend CPP is deriveable

Reply:
The way i set those up is that the labels contain the `origin` of the bug, not the backend in which it runs. Is this a meaningful  or useful differentiation? not sure. will remove for now